examples/lighting-app/k32w/BUILD.gn

```diff
@@ -72,8 +72,8 @@ k32w_executable("light_app") {
     "${chip_root}/src/lib",
     "${chip_root}/third_party/mbedtls:mbedtls",
     "${k32w_platform_dir}/app/support:freertos_mbedtls_utils",
-    "${openthread_root}:libopenthread-cli-ftd",
-    "${openthread_root}:libopenthread-ftd",
+    "${openthread_root}:libopenthread-cli-mtd",
```

Comment:
This isn't related to NFC. Could we make it a separate change?

Reply:
@doru91 ?

Reply:
> @woody-apple , could you please push the updated Docker image to the Docker repository? Otherwise the K32W CI for this PR will keep failing. See PR (#6226) that updates the Dockerfile.

Please note, the docker file fails builds. You can see an example here:
https://github.com/project-chip/connectedhomeip/runs/2453643151

Reply:
I've reverted back to the previous k32w for now

Reply:
Thanks, I fixed the compilation issues and updated to the latest k32w Docker image.

Reply:
> This isn't related to NFC. Could we make it a separate change?

Removed it from this PR. Will make a separate change.